aws/resource_aws_instance_test.go

```diff
@@ -3344,18 +3344,9 @@ data "aws_availability_zones" "current" {
 }
 
 func testAccInstanceConfigInDefaultVpcBySgName(rName string) string {
-	return testAccLatestAmazonLinuxHvmEbsAmiConfig() + fmt.Sprintf(`
-data "aws_availability_zones" "current" {
-  # Exclude usw2-az4 (us-west-2d) as it has limited instance types.
-  exclude_zone_ids = ["usw2-az4"]
```

Comment:
We probably need another static check that looks for these hardcoded excluded zone IDs. This PR continues the badness but is a step toward replacing them with `testAccAvailableEc2InstanceTypeForAvailabilityZone()`.

Reply:
Sure, #14390